### PR TITLE
Make total values in tooltip respect selected mix-mode

### LINKED ISSUE #5140: Incorrect axis labels in emissions chart (production mode)
**Describe the bug**
When using the emissions chart in production mode the y axis changes to an incorrect scale

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to the app
2. Select production mode
3. Open the country panel
4. Select carbon emissions
5. The carbon emissions chart will have incorrect labels for the Y axis

**Expected behavior**
The axis labels match the data

**Screenshots**
image

### LINKED ISSUE #5701: Production graph tooltip is showing total consumption GW instead of only production GW
**Describe the bug**
See screenshot 
IMG_5678

Chart shows a total power of about 0,6GW. However, the percentage calculation of solar (shown at 22%) uses a power of 1,66GW. 

**To Reproduce**
1. Go to Lithuania 
2. Scroll to the required hour 
3. Tap the solar power in chart 
4. 

**Expected behavior**
According to the chart, solar needs to be much higher than 22%

**Smartphone (if relevant):**
 - Device: iPhone 13 mini

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Issue

Closes #5701

Closes #5140, #5759

## Description

This makes the calculation in breakdown tooltips take the selected mode (production/consumption) instead of only using production.

### Preview

Screenshot 2023-08-22 at 22 33 17

Screenshot 2023-08-22 at 22 33 09

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: web/src/features/charts/hooks/useEmissionChartData.ts [hunk 1]
```diff
@@ -22,7 +21,7 @@ export function useEmissionChartData() {
         datetime,
         layerData: {
           emissions: tonsPerHourToGramsPerMinute(
-            getTotalElectricity(value, mixMode === Mode.CONSUMPTION)
+            getTotalElectricity(value, true, mixMode)
```
Comment (madsnedergaard) [inline web/src/features/charts/hooks/useEmissionChartData.ts:24]: This is the silly bug with emissions chart axis 🙈

File: web/src/features/charts/graphUtils.test.ts [hunk 2]
```diff
@@ -84,3 +90,51 @@ describe('getElectricityProductionValue', () => {
...
+  });
+
+  it('handles power for production', () => {
+    const actual = getTotalElectricity(zoneData, false, Mode.PRODUCTION);
+    expect(actual).toEqual(300);
+  });
+
+  it('returns NaN when productionValue is null', () => {
+    const actual = getTotalElectricity(
+      { ...zoneData, totalProduction: Number.NaN },
```
Comment (VIKTORVAV99) [inline web/src/features/charts/graphUtils.test.ts:126]: Will do a full review and testing tomorrow but until then:
Shouldn't this be Null if it tests that that it should return Nan if it's null? 😉
```suggestion
      { ...zoneData, totalProduction: Null},
```

**Review by tonypls — APPROVED** [review]
LGTM, lots of nice fixes with minimal changes

**Comment by madsnedergaard** [comment]
Good point about the text change @VIKTORVAV99, it requires updating all translations, so I'll address that later in a new PR :)

**Review by VIKTORVAV99 — APPROVED** [review]

**Pull request merged**